example/Dockerfile

```diff
@@ -1,10 +1,22 @@
-FROM alpine:latest
+# A Go base image is enough to build nitriding reproducibly.
+# We use a specific instead of the latest image to ensure reproducibility.
+FROM golang:1.20 as builder
 
-RUN mkdir -p /lib64 && ln -sf /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
-RUN apk add --no-cache py3-requests
+WORKDIR /
 
-COPY nitriding /
-COPY service.py /
-COPY start.sh /
+# Clone the repository and build the stand-alone nitriding executable.
+RUN git clone https://github.com/brave/nitriding.git
```

Comment:
Doesn't this make it harder to test local changes?

Reply:
It does but I'm fine with that considering that this is just an example application.